tools/packaging/static-build/kernel/build.sh

```diff
@@ -72,5 +73,6 @@ docker run --rm -i -v "${repo_root_dir}:${repo_root_dir}" \
 	--env DESTDIR="${DESTDIR}" --env PREFIX="${PREFIX}" \
 	--env USER="${USER}" \
 	--user "$(id -u)":"$(id -g)" \
+	--env KBUILD_SIGN_PIN="${KBUILD_SIGN_PIN}" \
```

Comment:
Put this before the --user, line, please.

Reply:
Done.